src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java

```diff
@@ -51,7 +52,7 @@
     @Test
     void should_create_case_if_classification_new_application_with_documents_and_ocr_data() throws IOException {
         setUpTransformation(getTransformationResponseBody("ok-no-warnings.json"));
-        setUpCcdSearchEmptyResult(getCcdResponseBody("search-result-empty.json"));
+        setUpCcdSearchResult(getCcdResponseBody("search-result-empty.json"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "search-result-empty.json" appears 7 times in this file; the first occurrence is on line 55](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4473817)